package.json

```diff
@@ -92,7 +92,11 @@
     "why-did-you-update": "1.0.6",
     "winston": "2.1.1"
   },
+  "optionalDependencies": {
+    "win-node-env": "0.4.0"
+  },
   "scripts": {
+    "start-nodemon": "nodemon app.js",
```

Comment:
what dis

Reply:
https://nodemon.io/

Nodemon is a utility that will monitor for any changes in your source and automatically restart your server. Perfect for development

Reply:
👍